### ISSUE TITLE
Install does not work

### ISSUE THREAD

**Issue by milasudril** [natecraddock/AddonDevelopmentTool#9]
I click install, but my Export addon does not appear in the export menu. Blender claims the addon has been installed, but it is not visible among addons in User Preferences either.

**Comment by natecraddock** [comment]
Does it work if you install without using the addon?

**Comment by milasudril** [comment]
Yes, it does. I guess I should post information about Blender also:

# Blender:

version: 2.77 (sub 0), branch: master, commit date: 2016-04-05 18:12, hash: abf6f08, type: RelWithDebInfo
build date: 2016-04-08, 19:09:13
platform: Linux
binary path: '/usr/lib/blender/blender'
build cflags:  -Wall -Wcast-align -Werror=implicit-function-declaration -Werror=return-type -Werror=vla -Wstrict-prototypes -Wmissing-prototypes -Wno-char-subscripts -Wno-unknown-pragmas -Wpointer-arith -Wunused-parameter -Wwrite-strings -Wlogical-op -Wundef -Winit-self -Wnonnull -Wmissing-include-dirs -Wno-div-by-zero -Wtype-limits -Wformat-signedness -Wuninitialized -Wredundant-decls -Wshadow -Wno-error=unused-but-set-variable  -fuse-ld=gold -fopenmp -std=gnu99  -msse2  -msse -pipe -fPIC -funsigned-char -fno-strict-aliasing
build cxxflags:  -Wredundant-decls -Wall -Wno-invalid-offsetof -Wno-sign-compare -Wlogical-op -Winit-self -Wmissing-include-dirs -Wno-div-by-zero -Wtype-limits -Werror=return-type -Werror=implicit-function-declaration -Wno-char-subscripts -Wno-unknown-pragmas -Wpointer-arith -Wunused-parameter -Wwrite-strings -Wundef -Wformat-signedness -Wuninitialized -Wundef -Wmissing-declarations  -fuse-ld=gold -fopenmp  -msse2  -msse -pipe -fPIC -funsigned-char -fno-strict-aliasing
build linkflags:  -Wl,--version-script='/build/blender-tc8YN5/blender-2.77-a-1459879952/source/creator/blender.map'
build system: CMake

# Python:

version: 3.5.1+ (default, Mar 30 2016, 22:46:26) 
[GCC 5.3.1 20160330]
paths:
    '/usr/share/blender/2.77/scripts/addons_contrib'
    '/home/torbjorr/.config/blender/2.77/scripts/addons'
    '/usr/share/blender/2.77/scripts/addons'
    '/usr/share/blender/2.77/scripts/startup'
    '/usr/share/blender/2.77/scripts/modules'
    '/usr/lib/python35.zip'
    '/usr/lib/python3.5'
    '/usr/lib/python3.5/plat-x86_64-linux-gnu'
    '/usr/lib/python3.5/lib-dynload'
    '/usr/local/lib/python3.5/dist-packages'
    '/usr/lib/python3/dist-packages'
    '/usr/share/blender/2.77/scripts/freestyle/modules'
    '/usr/share/blender/2.77/scripts/addons/modules'
    '/home/torbjorr/.config/blender/2.77/scripts/addons/modules'
